### ISSUE TITLE
Setup Errors

### ISSUE THREAD

**Issue by PaimonWheeWhee** [WeiWuxianYI/animemoon#3]
Ok, I managed to set it up, and when I did, I ran into this error. How can I solve this?
![Screen Shot 2021-05-22 at 5 07 39 PM](https://user-images.githubusercontent.com/81248955/119242024-3952b000-bb20-11eb-92d7-15a19beabce4.png)

**Comment by WeiWuxianYI** [comment]
What did you do exactly?

**Comment by PaimonWheeWhee** [comment]
That’s what happens when I first run the project, I didn’t make any changes yet

**Comment by WeiWuxianYI** [comment]
Seems like you can't connect to the database, have you checked permissions?

**Comment by PaimonWheeWhee** [comment]
Permissions for what

**Comment by WeiWuxianYI** [comment]
Database permissions.

**Comment by PaimonWheeWhee** [comment]
I didn't set up a database yet :o

**Comment by WeiWuxianYI** [comment]
> I didn't set up a database yet :o

You need a php server and import the database

**Comment by PaimonWheeWhee** [comment]
What database should I use

**Comment by WeiWuxianYI** [comment]
> What database should I use

Mysql

**Comment by PaimonWheeWhee** [comment]
Where do I put the database password and username and stuff like that?

**Comment by WeiWuxianYI** [comment]
Config.php

**Comment by theroshanme** [comment]
Ah i added username password but getting same issue as PaimonWheewhee any way to fix it

**Comment by WeiWuxianYI** [comment]
> Ah i added username password but getting same issue as PaimonWheewhee any way to fix it

You need to have Mysql database, did you import the database that was given to you in the files? If you haven't you got to import it. Also make sure database has all permissions.  

You can get a cheap hosting for 1-3$ for a month, test the website there. Or if you got discord we can talk and I'll do the setup for you

**Comment by theroshanme** [comment]
Yes,I have imported that and i added username as well as password anyway here is my discord : ItachiUchiya👑 #0263

**Comment by WeiWuxianYI** [comment]
> Yes,I have imported that and i added username as well as password anyway here is my discord : ItachiUchiya👑 #263

Last digit is missing i cant add you

**Comment by theroshanme** [comment]
Oops my bad its #0263

**Comment by PaimonWheeWhee** [comment]
When you still didn't include the last number moment 😂

**Comment by theroshanme** [comment]
wtf i am adding hash zero six three

**Comment by theroshanme** [comment]
but it didnt get attached 🤣 # 0263

**Comment by theroshanme** [comment]
so,you cannot add zero before hash ah

**Comment by Houcem777** [comment]
![Screenshot (23)](https://user-images.githubusercontent.com/85787101/121772504-a67cc280-cb6d-11eb-97d1-ef6ac95a973e.png)

**Comment by Houcem777** [comment]
Please help me tosolve this

**Comment by PaimonWheeWhee** [comment]
I got that too how strange

**Comment by WeiWuxianYI** [comment]
You must have a database password set,  make sure that database has every permission.

**Comment by Markowde3jani** [comment]
hello can i have your discord i need your help  i set up every thing and i get this when i try access to the admin 
![ddsssad](https://user-images.githubusercontent.com/88109300/127780939-b1c67613-d6ef-4cb9-915c-ae11b14a4f92.png)
i will set this in a online hosting and ill reach out to you if its fixed now only testing in a local

**Comment by Markowde3jani** [comment]
i get the same

**Comment by Markowde3jani** [comment]
and +this 

![aniiims](https://user-images.githubusercontent.com/88109300/127782415-64735a61-f4f6-482c-b154-eb8b2bd2b7fb.png)
